vms/avm/block/standard_block.go

```diff
@@ -88,5 +88,16 @@ func NewStandardBlock(
 		Time:         uint64(timestamp.Unix()),
 		Transactions: txs,
 	}
-	return blk, initialize(blk, cm)
```

Comment:
Calling `initialize` here used to call `blk.initialize` which would re-initialize the transactions. Now we explicitly do not re-initialize the transactions.